dist/tools/pyterm/pyterm

```diff
@@ -275,6 +275,17 @@ class SerCmd(cmd.Cmd):
                           "out" % line)
         for tok in line.split(';'):
             tok = self.get_alias(tok)
+            # handle ctrl+d nicely
+            if tok == "EOF":
+                quit = ""
+                while quit not in ('y', 'n'):
+                    quit = input('Exit pyterm ([y]/n)? ').lower()
```

Comment:
If `y` is the default, as marked by the `[y]`, then anything other than `n` should result into being interpreted as `y`.

Reply:
No sorry, that's me, I was still looking if I could do better.

But to answer your first question: the user has to answer either `y` of `n` or just press enter (since `y` is the default). For any other action, he's prompted again. This is the same behaviour with IPython

Reply:
OK fine.